data/formative-questionnaires/personal-social-and-emotional-development.yml

```diff
@@ -25,17 +25,15 @@ personal-social-and-emotional-development:
         multi_select: false
         label: |
           Read the scenario and answer the question:
-          Yusuf and Noah are playing together outside when Noah trips and bumps his knee,
-          which causes him to cry. Yusuf is quick to try to comfort Noah and looks for help from an adult.
-          Later, Yusuf recreates the situation with dolls and uses the opportunity
-          to express his worries about the accident.
+          Yusuf and Noah are playing together outside when Noah trips and bumps his knee, which causes him to cry. Yusuf is quick to try to comfort Noah and looks for help from an adult.
```

Comment:
There needs to be an extra line break between 'Read the scenario.....' and 'Yusuf and Noah'

Reply:
Fixed